README.md

```diff
@@ -17,122 +17,34 @@ Guidelines](CONTRIBUTING.md). While we prefer the creation of
-  
-- Make sure to link to the agenda at the beginning of the meeting (`Agenda: ...`)
-- Make sure the scribe is identified (`Scribe: ...` or `scribe+ ...` 
-  identifying someone else, or `scribe+` identifying oneself). Scribes, please 
-  familiarize yourself with general scribing guidance 
-  [here](https://www.w3.org/2008/04/scribe.html) and 
-  [here](https://www.w3.org/2008/xmlsec/Group/Scribe-Instructions.html).
-- Make sure topics are labeled when the topic changes (`Topic: ...`)
-- Make sure that action items are listed so that they can be added to issues 
-  later ("Action: ...")
-
-#### After the Meeting
-- Kick everyone from the meeting, and click **`Stop Recording`**
-- [Publish the minutes](https://github.com/w3c-ccg/traceability-interop/tree/main/docs/weekly-minutes)
-
-## [Open API Specification](https://w3c-ccg.github.io/traceability-interop/)
-
-The spec contains documentation on use cases as well as required and optional 
-API operations.
-
-### Included scripts for managing OpenAPI Specification
-
-1. `npm run validate-spec`: This command can be used to validate the 
-   `openapi.yml` spec, ensuring there are no issues.
-2. `npm run preserve`: This command is used to bundle our `openapi.yml` file 
-   into an `openapi.json` file that can be used to import the collection/spec.
-
-For additional documentation on how the `swagger-cli` can be used, visit 
-[here](https://www.npmjs.com/package/swagger-cli).
-
-## Importing the OpenAPI Specification Into Postman
-
-If you are using Postman, you can import all of the available API endpoints 
-into an easy to use collection by following the instructions below:
-
-1. While inside a Postman Workspace, you should see an **`import`** button at 
-   the top of your list of collections. Click this to begin the importing process.
-2. After clicking **`import`**, a dialog will open providing you with a couple 
-   of options for importing the collection. The easiest is to import using a 
-   **`Link`**. To do this, simply paste the following URL into the input box, 
-   and click **`Import`**: 
-   `https://w3c-ccg.github.io/traceability-interop/openapi/openapi.json`. 
-   Alternatively, you can use the **`Raw text`** option by copying the JSON 
-   found at 
-   `https://w3c-ccg.github.io/traceability-interop/openapi/openapi.json`, and 
-   pasting it into the input box.
-
-You should see something like this in your collections once you have 
-successfully imported the spec:
-
-<p align="center">
-  <a href="./docs/imported-collection.png">
-    <img src="./docs/imported-collection.png" width="25%" height="25%" align="center">
-  </a>
-</p>
+
+## Getting Started
+
+
+
+- [To generate a report](./reporting/README.md)
+- [To import the OpenAPI specification into Postman](./OPENAPI.MD)
+- For local development of the specification you will need to run the following: 
+```
+git clone git@github.com:w3c-ccg/traceability-interop.git
+cd traceability-interop
+npm i
+npm run serve
+```
```

Comment:
~~~suggestion
- For local development of the specification you will need to run the following: 
  ```
  git clone git@github.com:w3c-ccg/traceability-interop.git
  cd traceability-interop
  npm i
  npm run serve
  ```
~~~